concordium-std/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased changes
 
 - Set minimum Rust version to 1.65.
+- Introduce `block_time` for `HasChainMetadata` which should be used for contracts written on P6 and onwards.
```

Comment:
```suggestion
- Introduce a new method `block_time` as an alias to `slot_time` in the `HasChainMetadata` trait.
```

There is no reason for telling them to use it from p6 onward. There is nothing wrong with using slot_time since it is semantically equivalent.